### PR TITLE
node/repair: allow multiple sequences in a tx

### PR DESCRIPTION
**Pull request by kcsongor**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1183)

### CODE REVIEW

**Review by leoluk — APPROVED** [review]

**Review by SEJeff — APPROVED** [review]

File: node/hack/repair_terra/repair.go [hunk 1]
```diff
@@ -47,41 +47,42 @@ func getAdminClient(ctx context.Context, addr string) (*grpc.ClientConn, error,
...
-		return 0, fmt.Errorf("terra tx has no events")
+		return []uint64{}, fmt.Errorf("terra tx has no events")
 	}
 	msgs := EventsToMessagePublications(*terraAddr, txHash, events.Array())
 	// Should only ever be 1 message. Stole the above function from watcher.go
-	if len(msgs) != 1 {
-		return 0, fmt.Errorf("EventsToMessagePublications returned %d msgs", len(msgs))
+	var sequences = []uint64{}
+	for _, msg := range msgs {
+		sequences = append(sequences, msg.Sequence)
```
Comment (evan-gray) [inline node/hack/repair_terra/repair.go:83]: We should filter out emitters that don't match so we don't exit earlier on lower sequence numbers (or incorrectly match up the wrong one). @SEJeff to take a look 🙂

**Review by evan-gray — CHANGES_REQUESTED** [review]

File: node/hack/repair_terra/repair.go (hunk 1 above)
Comment (kcsongor) [reply node/hack/repair_terra/repair.go:83]: another thing we should do is make sure we only do a single reobservation request per transaction (even if it contains multiple sequence numbers)

**Review by evan-gray — APPROVED** [review]
Looks improved!

File: node/hack/repair_terra/repair.go (hunk 1 above)
Comment (SEJeff) [reply node/hack/repair_terra/repair.go]: I implemented your suggestion @evan-gray in c3ae93dab87015d810baea41bcdad89c3c5066bf.

We can do your suggestion in a followup @kcsongor, but what we have works perfectly.

**Review by SEJeff — APPROVED** [review]
Tested and verified working

File: node/hack/repair_terra/repair.go (hunk 1 above)
Comment (evan-gray) [reply node/hack/repair_terra/repair.go]: I agree we don’t need to change further right now 🙂

**Pull request merged**